common/customports/customport.go

```diff
@@ -28,6 +28,39 @@ func (c *CustomPorts) String() string {
 	return "Custom Ports"
 }
 
+// ValidateCustomPorts to validate the custom port range
+func ValidateCustomPorts(value string) {
```

Comment:
I think we should change the signature to `func ValidateCustomPorts(value string) error` and just return the errors here.

Reply:
Done.